src/polymer/create-element-extension.ts

```diff
@@ -57,15 +57,18 @@ class CreateElementExtension extends Rule {
               parsedDocument: document.parsedDocument,
               code: 'create-element-string-extension',
               severity: Severity.WARNING, sourceRange,
-              message: 'Element extension via the is string is deprecated.'
+              message: stripWhitespace(`
+                Element extension via the is string is deprecated.
```

Comment:
What about:

`document.createElement with a custom element tagname as the second parameter is deprecated.`